src/fetcher.js

```diff
@@ -516,7 +516,8 @@ var Fetcher = function Fetcher (store, timeout, async) {
     } else {
       // See https://www.iana.org/assignments/link-relations/link-relations.xml
       // Alas not yet in RDF yet for each predicate
-      predicate = kb.sym(Uri.join(rel, 'http://www.iana.org/assignments/link-relations/'))
+      /// encode space in e.g. rel="shortcut icon"
+      predicate = kb.sym(Uri.join(encodeURIComponent(rel), 'http://www.iana.org/assignments/link-relations/'))
```

Comment:
I agree that `NamedNode`s should url encode spaces in their values before serialization. But code that constructs named nodes should be providing URIs which resolve to real data.

Alternatively, I think we should have [`NamedNode.toString`](https://github.com/linkeddata/rdflib.js/blob/master/src/named-node.js#L55-L57) do the url encoding.